File: app/src/main/java/com/pavelprymak/propodcast/model/network/pojo/recommendations/LookingFor.java
package com.pavelprymak.propodcast.model.network.pojo.recommendations;

import com.squareup.moshi.Json;


public class LookingFor{

	@Json(name = "cross_promotion")
	private boolean crossPromotion;

	@Json(name = "sponsors")
	private boolean sponsors;

	@Json(name = "guests")
	private boolean guests;

	@Json(name = "cohosts")
	private boolean cohosts;

	public void setCrossPromotion(boolean crossPromotion){
		this.crossPromotion = crossPromotion;
	}

	public boolean isCrossPromotion(){
		return crossPromotion;
	}

	public void setSponsors(boolean sponsors){
		this.sponsors = sponsors;
	}

	public boolean isSponsors(){
		return sponsors;
	}

	public void setGuests(boolean guests){
		this.guests = guests;
	}

	public boolean isGuests(){
		return guests;
	}

	public void setCohosts(boolean cohosts){
		this.cohosts = cohosts;
	}

	public boolean isCohosts(){
		return cohosts;
	}

	@Override
 	public String toString(){
		return 
			"LookingFor{" + 
			"cross_promotion = '" + crossPromotion + '\'' + 
			",sponsors = '" + sponsors + '\'' + 
			",guests = '" + guests + '\'' + 
			",cohosts = '" + cohosts + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/pavelprymak/propodcast/model/network/pojo/podcasts/LookingFor.kt
package com.pavelprymak.propodcast.model.network.pojo.podcasts

import com.squareup.moshi.Json


data class LookingFor(

    @Json(name = "cross_promotion")
    var isCrossPromotion: Boolean = false,

    @Json(name = "sponsors")
    var isSponsors: Boolean = false,

    @Json(name = "guests")
    var isGuests: Boolean = false,

    @Json(name = "cohosts")
    var isCohosts: Boolean = false

)